graph/src/data/subgraph/mod.rs

```diff
@@ -198,7 +198,7 @@ pub struct UnresolvedMapping {
 }
 
 // Avoid deriving `Clone` because cloning a `Module` is expensive.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
As the comment notes, having `Clone` here is not good. Are you using this anywhere?

Reply:
Ah shoot, I should've noticed that comment!

Reply:
I'm going to rework the write methods to take the values by ref and clone internally, instead of cloning before passing the subgraph manifest to the write method